package.json

```diff
@@ -21,10 +21,11 @@
     "@angular/platform-browser-dynamic": "~9.0.5",
     "@angular/router": "~9.0.5",
     "@popperjs/core": "^2.0.6",
-    "angular-archwizard": "~6.0.0",
+    "angular-archwizard": "^6.0.0",
     "bootstrap": "^4.3.1",
     "font-awesome": "^4.7.0",
     "jquery": "^3.4.1",
+    "ngx-bootstrap": "^5.6.1",
```

Comment:
Do we absolutely require `ngx-bootstrap` here? I understand that it makes working with `bootstrap` easier, but is it absolutely necessary? Every additional dependency we add requires effort to ensure that it stays up to date and compatible with the other dependencies and the application itself. Therefore I would prefer if we try to keep the number of dependencies as low as possible :)

Reply:
I replaced ngx-bootstrap dropdown with bootstrap button toggle.